x-pack/plugins/infra/public/components/alerting/metrics/alert_flyout.tsx

```diff
@@ -35,6 +35,7 @@ export const AlertFlyout = (props: Props) => {
             },
             toastNotifications: services.notifications?.toasts,
             http: services.http,
+            docLinks: services.docLinks,
```

Comment:
Can we get `docLinks` documented here: https://github.com/elastic/kibana/blob/a9b8a621f7f75987796db28f27da8015a791bd30/x-pack/plugins/triggers_actions_ui/README.md#embed-create-alert-flyout-to-kibana-plugins

to make sure this dependency is clear?